src/App.css

```diff
@@ -1,4 +1,91 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Spartan:wght@400;500;600;700&display=swap');
+
+:root {
+  --white: #ffffff;
+  --dark-seafoam: #1aaf71;
+  --navy-blue: #001047;
+  --black: #000000;
+}
+
+:root {
+  --spacing-xxs: 0.25rem;
+  --spacing-xs: 0.5rem;
+  --spacing-s: 0.75rem;
+  --spacing-m: 1rem;
+  --spacing-l: 1.25rem;
+}
+
+.Spartan-62pt {
```

Comment:
I think we should not define the font CSS here, instead, we should define them in the `theme.ts` file, in the `typography` field, something like:

```
createMuiTheme({
  typography = {
    fontFamily: "Inter",
    h2: {
      fontSize: 20,
      fontWeight: 500,
    },
  },
   ....
};
```

Then, you will be able to do something like <Typography variant="h5" /> in order to access to that style.

Let me know if this doesn't makes sense

Reply:
I set styling to reference the material ui theme. The fonts and colors will look correct when the pull request for issue #9 is completed and merged.